README.md

```diff
@@ -47,11 +113,199 @@ Parameters:
 --no-upgrade             Don't automatically run `occ upgrade` when the Updater finishes (note: `occ upgrade` is required after Updater updates the application code in order to push out any database changes in the newly deployed version of Nextcloud)
 ```
 
+## How it works
+
+Keeping Nextcloud Server up-to-date, at a high level, consists of three things:
+
+1. Updating the code
+2. Upgrading the database
+3. Updating all shipped and locally deployed apps
+
+The Updater app handles step #1 (see [*What does the updater do?*](https://docs.nextcloud.com/server/latest/admin_manual/maintenance/update.html#what-does-the-updater-do)) and prepares the environment for steps #2 and #3 (which are handled by `occ upgrade` *after* the Updater successfully completes).
+
+The Updater app is designed to be as independent from Server as possible for reliability purposes.
```

Comment:
Not for reliability, but because it needs to run while the server is down because its code is being updated.

Reply:
Agreed. That's what I meant. :smile:  But you're right: I chose ambiguous wording. I've revised.